src/devices/tuya.ts

```diff
@@ -13334,6 +13334,101 @@ const definitions: DefinitionWithExtend[] = [
             ],
         },
     },
+    {
+        fingerprint: tuya.fingerprint('TS0601', ['_TZE204_eaulras5']),
+        model: 'PJ3201A',
+        vendor: 'Dongguan Pinjia Technology Co.,LTD.',
+        description: 'Human Presence Sensor',
+        fromZigbee: [tuya.fz.datapoints],
+        toZigbee: [tuya.tz.datapoints],
+        configure: tuya.configureMagicPacket,
+        exposes: [
+            e.presence(),
+            e.occupancy(),
+            e.numeric('target_dis_closest', ea.STATE).withDescription('the target distance away from the sensor').withUnit('m'),
+            e
+                .numeric('mov_delay', ea.STATE_SET)
+                .withValueMin(0)
+                .withValueMax(43200)
+                .withValueStep(1)
+                .withDescription('setting the delay time to keep moving status  (the time from moving -> None)')
+                .withUnit('s'),
+            e
+                .numeric('none_delay', ea.STATE_SET)
+                .withValueMin(0)
+                .withValueMax(43200)
+                .withValueStep(1)
+                .withDescription('setting the delay time to keep present status (the time from present -> None)')
+                .withUnit('s'),
+            e.illuminance(),
+            e
+                .numeric('far_mov_sensitivity', ea.STATE_SET)
```

Comment:
I've just tested this, it's mostly working / looking good but you need to update the reference to `far_mov_sensitivity` in the exposes after you renamed the datapoint.

same with target_distance_closest / target_dis_closest not matching


<img width="351" alt="image" src="https://github.com/user-attachments/assets/2f02c3c6-98d2-4482-9f00-ca289098b1d8" />

```suggestion
                .numeric('far_move_sensitivity', ea.STATE_SET)
```

Reply:
Renamed the keys completely.